gnomad/utils/filtering.py

```diff
@@ -225,26 +225,30 @@ def add_filters_expr(
 
 
 def subset_samples_and_variants(
-    mt: hl.MatrixTable,
+    t: Union[hl.MatrixTable, hl.vds.VariantDataset],
     sample_path: str,
     header: bool = True,
     table_key: str = "s",
     sparse: bool = False,
     gt_expr: str = "GT",
-) -> hl.MatrixTable:
+) -> Union[hl.MatrixTable, hl.vds.VariantDataset]:
     """
     Subset the MatrixTable to the provided list of samples and their variants.
 
-    :param mt: Input MatrixTable
+    :param t: Input MatrixTable or VariantDataset
```

Comment:
I guess we should think about the best param name for this since we use `t` to typically mean it will take a MatrixTable or a Table (I know this is what I put in the other file, but I just got thinking about it while reading this). I really don't know what is best, maybe `ds`? @klaricch thoughts?